Sources/SwiftProtobuf/Google_Protobuf_Any.swift

```diff
@@ -83,6 +83,9 @@ public extension Message {
   ///
   /// See `Google_Protobuf_Any.unpackTo()` for more discussion.
   ///
+  /// - Parameter unpackingAny: the message to decode.
+  /// - Throws: an instance of `AnyUnpackError`, `JSONDecodingError`, or
+  ///   `BinaryDecodingError` on failure.
   public init(unpackingAny: Google_Protobuf_Any) throws {
```

Comment:
fyi - https://github.com/apple/swift-protobuf/pull/356 could end up changing this.